examples/example_plugin/example_plugin/ui/NonModelView.jsx

```diff
@@ -0,0 +1,10 @@
+import GenericView from "@views/generic/GenericView";
+
+
+export default function NonModelView(){
+    return (
+        <GenericView gridBackground="white">
+            <div> Im a humble 🙈 Non Model Plugin View.</div>
```

Comment:
Let's use `app` instead of `plugin` for anything user-facing at this point please.